apps/agent/package.json

```diff
@@ -7,10 +7,10 @@
     "devchain": "buidler node",
     "build": "cd app && yarn build",
     "lint": "solium --dir ./contracts",
-    "test": "yarn ganache-cli:test",
-    "test:gas": "GAS_REPORTER=true yarn ganache-cli:test",
-    "coverage": "SOLIDITY_COVERAGE=true npm run ganache-cli:test",
-    "ganache-cli:test": "./scripts/ganache-cli.sh"
+    "test": "buidler test",
+    "test:gas": "REPORT_GAS=true buidler test --network localhost",
```

Comment:
Not sure, but I don't think the `localhost` network exists anymore in the buidler config?

Reply:
As we are using the gas reporter with the buidlerevm and they don't yet handle the start and stop of the chain. We need to manually ran `npx buidler node` and then connect to the chain with `--network localhost`. As explained on 
- https://github.com/cgewecke/buidler-gas-reporter/tree/master#using-buidlerevm-warning
- https://buidler.dev/buidler-evm/#connecting-to-buidler-evm-from-wallets-and-other-software

Reply:
I include a comment to give more context about it: https://github.com/aragon/aragon-apps/pull/1131/commits/11146312c843878283e9168efafcb4c3f6b85e75#diff-31b24dc429d79c5898460ef28314ee00R38

Reply:
Gotcha, so the `--network local` flag would tell it to connect to the separately run `buidlerevm` instance?

Reply:
Yes. The `buidlerevm` chain starts a JSON-RPC and WebSocket server and exposes it at `http://localhost:8545`. Then the buidler default configuration object has the following format:
```
{
    localhost: {
      url: "http://127.0.0.1:8545"
    },
    buidlerevm: {
      // See its defaults
    }
}
```

That's why you can use `--network localhost`. Maybe we should be explicit about `localhost` network?